mapviz_plugins/src/odometry_plugin.cpp

```diff
@@ -278,25 +278,70 @@ namespace mapviz_plugins
       PrintInfo("OK");
     }
   }
+  
+
+  void OdometryPlugin::Paint(QPainter* painter, double x, double y, double scale)
+  {
+    int interval = ui_.show_timestamps->value();
+    if (interval == 0)
+      return;
+
+    QTransform tf = painter->worldTransform();
+    QFont font("Helvetica", 10);
+    painter->setFont(font);
+    painter->save();
+    painter->resetTransform();
+
+    //set the draw color for the text to be the same as the rest
+    QPen pen(QBrush(ui_.color->color()), 1);
+    painter->setPen(pen);
+
+    std::list<StampedPoint>::iterator it = points_.begin();
+    int i = 0;//used to alternate between rendering text on some points
+    for (; it != points_.end(); ++it)
+    {
+      if (it->transformed && i++ % interval == 0)
+      {
+        QPointF point = tf.map(QPointF(it->transformed_point.getX(),
+                                       it->transformed_point.getY()));
+		QString time;
```

Comment:
It appears your editor has accidentally added some tab characters here. Files in `swri-robotics` use spaces instead of tabs, so it is good to match the existing style.

Replace tabs with two spaces each.